browser/js/identities.js

```diff
@@ -0,0 +1,140 @@
+// All of our identity services and matchers are defined here.
```

Comment:
These were mostly consolidated/moved from other places to allow for sharing of the matching code between the background scripts and the content injection scripts.